index.html

```diff
@@ -19,7 +19,7 @@
         />
       </div>
       <nav id="nav-bar">
-        <a href="#features" class="nav-link">Features</a>
+        <a href="#features" class="nav-link">Features </a>
         <a href="#video" class="nav-link">How It Works</a>
         <a href="#pricing" class="nav-link">Pricing</a>
       </nav>
```

Comment:
It would be great to use semantic HTML here, you can put `a` tags inside `ul` list.

Reply:
- Great job @OTMANE-ECHCHAFYKY However, I will suggest that you make use of polite language when giving reviews so it doesn't sound like you're giving orders to the student :+1: 
- These are polite suggestions: `"I think it would be better to use semantic tags", I will suggest...., Kindly consider...... etc`
- Kindly ensure that you add the reasons why a particular change is requested. You might know why you're requesting the change but the student might not know why so to avoid confusion, it is always advisable to add "Why" a change is requested :+1: 
- [ OPTIONAL ]  will suggest that you include emojis in your reviews so it sends a friendly tone to the recipient. :100: